network/manager.go

```diff
@@ -710,21 +712,21 @@ func cnsEndpointInfotoCNIEpInfo(endpointInfo restserver.EndpointInfo, endpointID
 		epInfo.IfName = ifName
 		epInfo.HostIfName = ipInfo.HostVethName
 		epInfo.HNSEndpointID = ipInfo.HnsEndpointID
+		epInfo.HNSNetworkID = ipInfo.HnsNetworkID
+		epInfo.MacAddress = net.HardwareAddr(ipInfo.MacAddress)
 	}
 	return epInfo
 }
 
+// generateCNSIPInfoMap generates a CNS ifNametoIPInfoMap structure based on CNI endpoint
 func generateCNSIPInfoMap(ep *endpoint) map[string]*restserver.IPInfo {
```

Comment:
im curious about why we are no longer looping through all secondary interfaces and adding them to the map, what has changed?

Reply:
This will be added with another PR by @QxBytes since he is working on Swift 2.0 changes and couple of refactores related to epInfo